csvcubed/csvcubed/writers/skoscodelistwriter.py

```diff
@@ -145,11 +147,12 @@ def _get_csvw_table_schema(self) -> dict:
     def _get_csvw_metadata(self) -> dict:
         scheme_uri = self._new_uri_helper.get_scheme_uri()
         additional_metadata = self._get_catalog_metadata(scheme_uri)
+        url = self._new_uri_helper._get_identifier_for_document()
 
         return {
             "@context": "http://www.w3.org/ns/csvw",
             "@id": scheme_uri,
-            "url": self.csv_file_name,
+            "url": url,
```

Comment:
This needs to point at the true path of the CSV file, not the URI that we are using as an identifier for it.

Also on a side-note: `_get_identifier_for_document` is (by convention) a private function since it starts with a `_`.